File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("height: ");
    }

    while (height <= 0 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height; j++)
        {
            if (i + j < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("\n");

    }
}

File: pset6/sentimental-mario-less/mario.py
from cs50 import get_int
import sys

# create a variable called 'height'
height = 0

# if the user fails to provide an integer between 1 and 8, re-prompt for the same again
while height < 1 or height > 8:
    height = get_int("height: ")

# print the pyramid
for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1))
